integration/commands_administration_test.go

```diff
@@ -56,27 +56,10 @@ func TestCommandsAdministrationCreateDropList(t *testing.T) {
 	err = db.Collection(name).Drop(ctx)
 	require.NoError(t, err)
 
-	// drop manually to check error
-	var res bson.D
-	err = db.RunCommand(ctx, bson.D{{"drop", name}}).Decode(&res)
-	expectedErr := mongo.CommandError{
-		Code:    26,
-		Name:    "NamespaceNotFound",
-		Message: `ns not found`,
-	}
-	AssertEqualCommandError(t, expectedErr, err)
-
 	err = db.CreateCollection(ctx, name)
 	require.NoError(t, err)
 
-	err = db.CreateCollection(ctx, name)
```

Comment:
A couple of these deleted tests seemed to be meaningful (checking for drop of nonexistent database and checking for creation of collection that already exists). Did you delete them because they are duplicates of some other tests? Or were they moved somewhere?

Reply:
Ah, I got it! Some logic changed, and these are not errors anymore... Still, then we should check that those are not errors then :)

Reply:
Lines 53–57 seems to do exactly that, but maybe we need to rewrite it more clearly. What do you think?